src/System.AppContext/tests/AppContext.Switch.cs

```diff
@@ -0,0 +1,136 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Xunit;
+public class AppContextSwitchesTests
+{
+    private static Random randomNumbers = new Random((int)DateTime.Now.Ticks);
+
+    [Fact]
+    public void SwitchNotFound()
+    {
+        string switchName = GetRandomSwitchName();
+        bool isEnabled;
+        Assert.False(AppContext.TryGetSwitch(switchName, out isEnabled));
+    }
+
+
+    [Fact]
+    public void EnableSwitch()
+    {
+        string switchName = GetRandomSwitchName();
+        AppContext.SetSwitch(switchName, true);
+
+        bool isEnabled;
+        Assert.True(AppContext.TryGetSwitch(switchName, out isEnabled) && isEnabled == true);
```

Comment:
Consider separating these into two separate Assert.True calls so that you know which part failed. (Same for the other occurrences of this pattern).